### PR TITLE
Separated logic of Sign In and Registration forms

### PR DESCRIPTION
**Pull request by AArlauskas**
Created separate RegistrationViewModel, fixed naming

### CODE REVIEW

File: Moodis/Moodis/Feature/Login/Register/RegisterForm.cs [hunk 1]
```diff
@@ -13,7 +13,7 @@ namespace Moodis.Feature.Login.Register
 {
     public partial class RegisterForm : Form
     {
-        LoginViewModel loginViewModel = new LoginViewModel();
+        RegisterViewModel loginViewModel = new RegisterViewModel();
```
Comment (tmaciulis22) [inline Moodis/Moodis/Feature/Login/Register/RegisterForm.cs]: `loginViewModel`

**Review by tmaciulis22 — CHANGES_REQUESTED** [review]

File: Moodis/Moodis/Feature/Login/Register/RegisterForm.cs (hunk 1 above)
Comment (AArlauskas) [reply Moodis/Moodis/Feature/Login/Register/RegisterForm.cs]: fixed

File: Moodis/Moodis/Feature/Menu/MenuForm.cs [hunk 2]
```diff
@@ -51,7 +51,7 @@ public async void UpdateLabels()
                     counter++;
                 }
                 menuViewModel.UserAddImage();
-                Console.WriteLine(LoginViewModel.currentUser.imageStats);
+                Console.WriteLine(SignInViewModel.currentUser.imageStats);
```
Comment (Pilkas-git) [inline Moodis/Moodis/Feature/Menu/MenuForm.cs]: Unnecessary writeline.

**Review by Pilkas-git — CHANGES_REQUESTED** [review]

File: Moodis/Moodis/Feature/Menu/MenuForm.cs (hunk 2 above)
Comment (AArlauskas) [reply Moodis/Moodis/Feature/Menu/MenuForm.cs]: Not my code, but fixed it 🍺

File: Moodis/Moodis/Feature/Login/Register/RegisterForm.cs [hunk 3]
```diff
@@ -23,7 +23,7 @@ public partial class RegisterForm : Form
         public RegisterForm()
         {
             InitializeComponent();
-            var loginViewModel = new LoginViewModel();
+            var registerViewModel = new RegisterViewModel();
```
Comment (tmaciulis22) [inline Moodis/Moodis/Feature/Login/Register/RegisterForm.cs]: Lol, why are we creating a new viewmodel here, when we have already created one in 16 line?

**Review by tmaciulis22 — CHANGES_REQUESTED** [review]

File: Moodis/Moodis/Feature/Login/Register/RegisterForm.cs (hunk 3 above)
Comment (tmaciulis22) [reply Moodis/Moodis/Feature/Login/Register/RegisterForm.cs]: Just remove initialization from 16 line, and on this line remove `var`, so this way constructor will create and assign the viewmodel to the property

File: Moodis/Moodis/Program.cs [hunk 4]
```diff
@@ -11,15 +11,45 @@ namespace moodis
...
+            {
+                //Application.Exit();
+                MessageBox.Show(WarningInRequest);
+                return;
+            }
+            
+            
+        }
+
+        private static bool checkForInternet()
```
Comment (tmaciulis22) [inline Moodis/Moodis/Program.cs:35]: This should work without `static`

File: Moodis/Moodis/Feature/Login/Register/RegisterForm.cs (hunk 3 above)
Comment (AArlauskas) [reply Moodis/Moodis/Feature/Login/Register/RegisterForm.cs]: Very good question ⚰️

File: Moodis/Moodis/Program.cs (hunk 4 above)
Comment (AArlauskas) [reply Moodis/Moodis/Program.cs:35]: Plot twist:


















It does not 😮

Comment (tmaciulis22) [reply Moodis/Moodis/Program.cs:35]: Oh right, Main is also static. my bad

File: Moodis/Moodis/Program.cs [hunk 5]
```diff
@@ -11,15 +11,45 @@ namespace moodis
...
+                Application.SetCompatibleTextRenderingDefault(false);
+                Application.Run(new LoginForm());
+            }
+            else
+            {
+                //Application.Exit();
+                MessageBox.Show(WarningInRequest);
+                return;
+            }
+
```
Comment (tmaciulis22) [inline Moodis/Moodis/Program.cs]: Found some empty lines :)))))))

File: Moodis/Moodis/Program.cs (hunk 4 above)
Comment (AArlauskas) [reply Moodis/Moodis/Program.cs:35]: This class starts the program and must be static, so we cannot change it

File: Moodis/Moodis/Program.cs [hunk 6]
```diff
@@ -11,15 +11,45 @@ namespace moodis
 {
     static class Program
     {
+        private const string WarningInRequest = "Connection failed. Is your internet turned on ?";
         /// <summary>
         /// The main entry point for the application.
         /// </summary>
         [STAThread]
+
```
Comment (tmaciulis22) [inline Moodis/Moodis/Program.cs]: Also empty line :))))))

File: Moodis/Moodis/Program.cs [hunk 7]
```diff
@@ -11,15 +11,45 @@ namespace moodis
...
+            if(checkForInternet())
+            {
+                Application.EnableVisualStyles();
+                Application.SetCompatibleTextRenderingDefault(false);
+                Application.Run(new LoginForm());
+            }
+            else
+            {
+                //Application.Exit();
```
Comment (tmaciulis22) [inline Moodis/Moodis/Program.cs]: Remove commented out code

File: Moodis/Moodis/Program.cs (hunk 5 above)
Comment (AArlauskas) [reply Moodis/Moodis/Program.cs]: Triggered

**Review by tmaciulis22 — APPROVED** [review]
Good job cleaning this mess 👏

**Pull request merged**